src/components/dashboard/index.js

```diff
@@ -238,7 +284,11 @@ const Dashboard = (props) => {
                     }}
                 />
             )}
-            {isLoading ? <DashboardSkeleton /> : componentContent}
+            {isLoading || analysisLoading ? (
```

Comment:
Looks like `isLoading` was updated to include `analysisLoading`, so this can just check `isLoading`